src/api/tryGetFileTemporaryUrl.js

```diff
@@ -1,11 +1,25 @@
 /* @flow strict-local */
 import type { Auth } from './apiTypes';
+import { tryParseUrl } from '../utils/url';
 import getFileTemporaryUrl from './messages/getFileTemporaryUrl';
 
 /**
  * Like getFileTemporaryUrl, but on error returns null instead of throwing.
+ *
+ * Validates that `href` parses to a URL object with auth.realm.
+ *
+ * This performs a call to `new URL` and therefore may take a fraction of a
+ * millisecond.  Avoid using in a context where it might be called more than
+ * 10 or 100 times per user action.
```

Comment:
Makes sense. In my next revision, I've also skipped it on `messageLinkPress`, because that function also operates on long time scales, like when it re-narrows, causing a navigation animation.

Reply:
Same with `handleImage`, which opens the lightbox.